llama_index/vector_stores/google/generativeai/genai_extension.py

```diff
@@ -0,0 +1,534 @@
+"""Temporary high-level library of the Google GenerativeAI API.
+
+The content of this file should eventually go into the Python package
+google.generativeai.
+"""
+
+import datetime
+import logging
+import re
+import uuid
+from dataclasses import dataclass
+from typing import Any, Dict, Iterator, List, MutableSequence, Optional
+
+import google.ai.generativelanguage as genai
+from google.api_core import client_options as client_options_lib
+from google.api_core import gapic_v1
+from google.protobuf import timestamp_pb2
+
+import llama_index
+
+_logger = logging.getLogger(__name__)
+_DEFAULT_API_ENDPOINT = "autopush-generativelanguage.sandbox.googleapis.com"
```

Comment:
leaving this as a comment so we don't forget - change this to prod

Reply:
I see. I'll wait for the final decision what that endpoint is.

Reply:
Changed to generativelanguage.googleapis.com